.github/workflows/release.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           TARBALL_PATH="${PROJECT_NAME}-v${PROJECT_VERSION}-${{ matrix.job.target }}${{ matrix.job.package-extension }}"
           echo "TARBALL_PATH=${TARBALL_PATH}" >> "${GITHUB_OUTPUT}"
 
-          if [ "${{ matrix.job.os }}" = "windows-2019" ]; then
```

Comment:
The build was broken because windows version was updated to windows-2022.

https://github.com/ikanago/omekasy/blob/5f526dc1e10742656d178cff773d272124513599/.github/workflows/release.yml#L37
We should use the regular expression, then the build wouldn't be broken even if windows version will be changed.

Reply:
# Test

```console
$ [[ "windows-2022" =~ ^windows- ]]; echo $?
0

$ [[ "ubuntu-latest" =~ ^windows- ]]; echo $?
1
```